_posts/2018-08-13-Introducing....md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: ### Introducing...
+categories: blog
+excerpt: ### blog introducing three people from Kirstie's lab 
+tags: [] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link: 
+date: 2019-08-13 ### UPDATE date
+modified:
+share: true
+author: GeorgiaHCA ###
+---
+
+:tada: :star2: :rocket: Introducing three of my stellar Turing colleagues... 
```

Comment:
I feel like this needs to say a little bit about why you're doing this blog post? Fine if you'd rather just jump in though. Let me know what you think.

Reply:
Yeah I think you're right! It's a bit alarm - added a small extra introduction to the introductions :)